libraries/AP_Baro/AP_Baro.h

```diff
@@ -177,6 +180,8 @@ class AP_Baro
         float altitude;                 // calculated altitude
         AP_Float ground_pressure;
         float p_correction;
+        uint8_t can_manager;
+        uint8_t uavcan_node;
```

Comment:
It still doesn't make sense to me having this here, it's completely unscalable - when other protocols are added we just keep adding fields that 90% of drivers don't need?
Also, why do UAVCAN instances need to know about each other? Doesn't each manager just need to know if it has registered that baro node? Unless I'm missing something that's what I see in current code but done in AP_Baro instead of AP_UAVCAN side.

By the way, this is only needed because of the hot-plugging situation right?